.github/workflows/_check-release.yml

```diff
@@ -1,6 +1,7 @@
 name: Release Check
 
 on:
+  workflow_dispatch:
```

Comment:
missing inputs, this will crash I believe without them

Reply:
Workflow dispatch doesn't take any inputs. It's exactly the same situations as in `website-deploy` and `nightly` workflows.

Reply:
But this workflow actually has `${{ inputs.* }}` expressions, and I believe actions runner will yell that it doesn't know these inputs, when executed via dispatch.

Reply:
Yeah, I don't think it would crash, but it is possible that it would complain about it so it makes sense to specify those.